### DISCUSSION TITLE
PR's closed by bots

### DISCUSSION THREAD

**Discussion by gabsmprocha** [community/community#26305] [Actions]
<p>Hello everyone!</p>
<p>When a PR is closed through a bot for example, it doesn’t run the workflow, any ideas on how to solve this?</p>

**Comment by airtower-luna** [comment]
<p>What kind of bot are you talking about? Code running in an Actions workflow, or something else?</p>

**Comment by gabsmprocha** [comment]
<p>a bot that analyzes PR’s that are stale and closes</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/actions/stale/actions">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/actions/stale/actions" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/90699433/181096778-37bfe95c-e85f-4d33-9df3-220b6bac9816.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f64644fec84630f8e945e9800d0053734790b43_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f64644fec84630f8e945e9800d0053734790b43_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/f/1f64644fec84630f8e945e9800d0053734790b43.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/f/1f64644fec84630f8e945e9800d0053734790b43_2_10x10.png"></div>

<h3><a href="https://github.com/actions/stale/actions" target="_blank" rel="noopener nofollow ugc">Actions · actions/stale</a></h3>

  <p>Marks issues and pull requests that have not had recent interaction - Actions · actions/stale</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>That’s an <em>action</em>, which will appear as the Actions bot in the log because by default it uses the <code>GITHUB_TOKEN</code> for authorization. And any activity authorized by the <code>GITHUB_TOKEN</code> won’t trigger further workflows, <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow">as documented</a>.</p>
<p>If you want to trigger workflows, you need to <a href="https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token">create a PAT</a>, store that <a href="https://docs.github.com/en/actions/security-guides/encrypted-secrets">as a secret</a>, and provide it to the action that closes the PRs.</p>

**Comment by gabsmprocha** [comment]
<p><a class="mention" href="/airtower-luna">@airtower-luna</a> Thanks for the suggestion, I will apply it</p>

**Comment by gabsmprocha** [comment]
<p>it seems that creating a pat and using it didn’t work either )=</p>
